akka-persistence-query/src/main/mima-filters/2.5.25.backwards.excludes

```diff
@@ -0,0 +1,2 @@
+# #27411 changes not caught by previous mima versions
+ProblemFilters.exclude[IncompatibleSignatureProblem]("*")
```

Comment:
what implication will this have for finding future incompatibilities?

Reply:
It will not be any worse than with the previous version of mima, since `IncompatibleSignatureProblem` was added in 0.5.0.

It will miss any signature changes with 2.5.25 and earlier, but catch those still present between master and any future version.

I'll see if moving the `*` exclusion to 2.5.24, so we can take advantage of the `IncompatibleSignatureProblem` check earlier